src/sentry/conf/server.py

```diff
@@ -2504,6 +2505,8 @@ def build_cdc_postgres_init_db_volume(settings):
     KAFKA_INGEST_REPLAYS_RECORDINGS: {"cluster": "default"},
     # Metrics Testing Topics
     KAFKA_SNUBA_GENERICS_METRICS_CS: {"cluster": "default"},
+    # User Ip Events -- bucketed by region name
```

Comment:
```suggestion
    # Region to Control Silo messaging - eg. User Ip Events; bucketed by region name
```
Might be more future proof :shrug: